pywr/parameters/parameters.py

```diff
@@ -109,7 +109,17 @@ def load(cls, model, data):
         x = np.array(data.pop("x"))
         y = np.array(data.pop("y"))
         kind = data.pop("kind", "linear")
-        return cls(model, parameter, x, y, interp_kwargs={'kind': kind})
+        interp_kwargs = data.pop("interp_kwargs", None)
+        if interp_kwargs:
+            interp_kwargs["kind"] = kind
+            if "fill_value" in interp_kwargs and isinstance(interp_kwargs["fill_value"], list):
```

Comment:
Could this be a static method on the InterpolatedParameter base class?

Reply:
I've added a property for inter_kwargs on the base class